src/store/profile.js

```diff
@@ -137,8 +145,31 @@ export const useProfileStore = defineStore({
 
       const profile = await this.getProfile(id)
       this.setActiveProfile(profile)
+      await this.loadProfileThreads()
 
       return profile
+    },
+    /**
+     * Fetches the threads from user with a particular feedId
+     */
+    async loadProfileThreads () {
+      const res = await apolloClient.query({ query: GET_PROFILE_THREADS, variables: { id: this.activeProfile.id, cursor: null } })
+      if (res.errors) throw res.errors
+
+      this.$patch((state) => {
```

Comment:
:eyes: I haven't used $patch in pinia, is this needed here?
I think I've only ever done `state.threads = res.data.getThreads` directly